admission.bundle.yaml

```
@@ -106,7 +106,7 @@ spec:
             scheme: HTTPS
       initContainers:
       - name: admin-init
-        image: redislabs/operator-internal:6.0.6-6
+        image: redislabs/operator-internal:6.0.6-11
```

```suggestion
        image: redislabs/operator:6.0.6-11
```
Surprised we missed these last time, why did we have all these internals?

---

This file was left out on the list of files that are replaced, thank you for catching that.